docs/source/tutorials/datalab.ipynb

```diff
@@ -0,0 +1,1393 @@
+{
```

Comment:
@huiwengoh how about we name this file: `datalab_quickstart.ipynb` ?

Reply:
Renamed!